torch/csrc/jit/tensorexpr/loopnest.h

```diff
@@ -32,13 +32,13 @@ class TORCH_API LoopNest {
   // they can be deduced.
   LoopNest(
       Stmt* stmt,
+      const std::unordered_set<const Buf*>& input_bufs,
```

Comment:
Can you explain this change? This doesn't seem related. Removing buf_initializers in particular seems somewhat suspicious.

Reply:
I had to add `input_bufs` to get my tests to generate `LoopNest` from arbitrary `For` statement. I was wrong in removing `buf_initializers` as well. Now, I managed the tests without making any changes here. Reverted both the changes.